### ISSUE TITLE
Vuetify 3 issue: Dropdown Menu

### ISSUE THREAD

**Issue by joshvanallen** [SAP/inquirer-gui#651]
This is meant to be kept around as a documentation item. Currently the features that depend on Vuetify's Autocomplete feature (list and autocomplete) has an menu position calculation issue.  ([Related Vuetify issue](https://github.com/vuetifyjs/vuetify/issues/17126))

**Comment by veredcon** [comment]
Good idea to open that issue for documentation and tracking! Thank you!

**Comment by vbangelov** [comment]
Issue should be fixed in latest Vuetify 3.8.1 introduced in https://github.com/SAP/inquirer-gui/pull/729. Closing as fixed.
